### PR TITLE
storage: optimize large azure chunked upload (PROJQUAY-3753)

### PR DESCRIPTION
**Pull request by kleesc**
- Increase nginx send timeout on blobs endpoints
- Reduce Azure blob block size

TODO: Spread block uploads over multiple worker threads.

### CODE REVIEW

File: storage/azurestorage.py [hunk 1]
```diff
@@ -232,41 +233,41 @@ def stream_upload_chunk(self, uuid, offset, length, in_fp, storage_metadata, con
         total_bytes_written = 0
 
         while True:
-            current_length = length - total_bytes_written
             max_length = (
-                min(current_length, self._max_block_size)
+                min(length - total_bytes_written, self._max_block_size)
```
Comment (bcaton85) [inline storage/azurestorage.py:237]: should `self._max_block_size` here also be set to the smaller block size (`1024 * 1024 * 128`)?

Comment (kleesc) [reply storage/azurestorage.py]: Updated to use the minimum of that value or what the api supports.

**Review by bcaton85 — APPROVED** [review]
LGTM

**Pull request merged**